src/utils/next.ts

```diff
@@ -104,8 +105,7 @@ export const scaffold = (wrapped : ScaffoldedGetServerSideProps, options? : Scaf
         };
 
         try {
-            const user = await ctx.z.resource('users', 'me').get();
-            augmentProps(user.data.data as ZetkinUser);
+            augmentProps(ctx.user as ZetkinUser);
```

Comment:
This doesn't look right. The `try` used to be there to verify that the user could be retrieved. If it couldn't, the `catch` would run `augmentProps(null)`. Now, you are not doing anything here that should raise an exception, so the `try` is no longer relevant.

The typecast should also not be necessary, as `ctx.user` should be typed correctly.

Reply:
True, sorry I missed this, looking at it now it's obvious.